scripts/L1_processPointClouds.R

```diff
@@ -53,7 +61,7 @@ clusterEvalQ(cl, library(terra))
 clusterEvalQ(cl, library(lidR))
 clusterExport(cl, c("gridInfo", "catObj", "dirPath"))
 parSapply(cl, 1:nrow(gridInfo), standardizePC, gridInfo, catObj, overlap=30, 
-          dirPath, type="align")
+           ROI=bciBorder, dirPath, type="align")
```

Comment:
Should the ROI argument go at the end to match the order of the function definition?

Reply:
Yes, good catch!